gpu/command_buffer/client/mapped_memory.h

```diff
@@ -88,6 +93,12 @@ class GPU_EXPORT MemoryChunk {
   //   token: the token value to wait for before re-using the memory.
   void FreePendingToken(void* pointer, unsigned int token) {
     allocator_.FreePendingToken(pointer, token);
+#if defined(CASTANETS)
+    if (std::string("renderer") ==
+      base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII("type")) {
+      mojo::SyncSharedMemoryHandle(shm_->backing()->GetGUID(), 0, GetLargestFreeSizeWithWaiting());
```

Comment:
Check offset size.

Reply:
Done.

Reply:
Does this always sync data from the beginning of the buffer?

Reply:
Sorry, I modificated this line checking the offset, but it seems it was not applied. I'll retry it.